CODE_OF_CONDUCT.md

```diff
@@ -5,7 +5,7 @@
 In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to make participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, sex characteristics, gender identity and expression,
+size, disability, ethnicity, sex characteristics, gender identity, and expression,
```

Comment:
"gender identity and expression" is one phrase, so we do not need to add a comma here.